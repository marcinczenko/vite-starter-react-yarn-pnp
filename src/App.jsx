const App = () => (
  <div className='w-screen h-screen flex flex-col items-center justify-center bg-blue-700 text-white text-4xl'>
    <p>vite-tailwind-starter</p>
    <div className='bg-white p-1'>
      <p className='text-xl text-blue-700'>with yarn pnp</p>
    </div>
    <p className='text-base'>with eslint</p>
    <p className='text-xl'>with local <span className='font-roboto-mono text-3xl font-thin'>Roboto Mono</span> font</p>
    <p className='text-xl'>and <span className='font-roboto-mono italic text-3xl font-thin'>Roboto Mono Italic</span></p>
  </div>
)

export default App
