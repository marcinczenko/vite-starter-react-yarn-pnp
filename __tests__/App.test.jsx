import { render, screen } from '@testing-library/react'

import App from '../src/App'

describe('App', () => {
  it('has proper styling', async () => {
    const { container } = render(<App />)
    expect(container).toMatchSnapshot()
  })

  it('displays welcome text', () => {
    render(<App />)
    expect(screen.getByText(/^vite-tailwind-starter/i)).toBeInTheDocument()
  })
})
