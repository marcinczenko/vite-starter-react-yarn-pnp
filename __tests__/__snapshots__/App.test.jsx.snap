// Vitest Snapshot v1

exports[`App > has proper styling 1`] = `
<div>
  <div
    class="w-screen h-screen flex flex-col items-center justify-center bg-blue-700 text-white text-4xl"
  >
    <p>
      vite-tailwind-starter
    </p>
    <div
      class="bg-white p-1"
    >
      <p
        class="text-xl text-blue-700"
      >
        with yarn pnp
      </p>
    </div>
    <p
      class="text-base"
    >
      with eslint
    </p>
    <p
      class="text-xl"
    >
      with local 
      <span
        class="font-roboto-mono text-3xl font-thin"
      >
        Roboto Mono
      </span>
       font
    </p>
    <p
      class="text-xl"
    >
      and 
      <span
        class="font-roboto-mono italic text-3xl font-thin"
      >
        Roboto Mono Italic
      </span>
    </p>
  </div>
</div>
`;
